README.md

```diff
@@ -129,7 +190,21 @@ SQL database. If you need an example, you can use `people` and `wishes` models.
 Your answer...
 
 ```
-Replace this with your answer
+Schema - is the organization or structure for a database.
```

Comment:
The rows are going to be instances of each table, so the schema won't be creating those. That's either you or a seed file.